apps/judicial-system/web/src/routes/Prosecutor/Indictments/Overview/Overview.tsx

```diff
@@ -68,6 +74,8 @@ const Overview: React.FC = () => {
       activeSubSection={
         caseHasBeenSentToCourt
           ? undefined
+          : features.includes(Feature.CASE_FILE_ROUTE)
+          ? 4
```

Comment:
Consider using a defined constant instead of number.

Reply:
This is just a temporary thing while we are using a feature flag and will be removed when we remove the flag.